hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/streams/InterruptibleRunnable.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.utilops.streams;
+
+public interface InterruptibleRunnable {
+    void run() throws InterruptedException;
```

Comment:
Now that I've seen `RecordAssertions` why not provide a default. method here that encapsulates an abstract `run()` with the try/catch/rethrow something better?